.github/workflows/tests-windows.yaml

```diff
@@ -0,0 +1,79 @@
+name: Test Windows
+
+on: [push, pull_request, merge_group]
+
+concurrency:
+  # Cancel previous workflows of the same PR, but only for PRs
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+env:
+  CC: clang
+  CXX: clang
+
+jobs:
+  test_skimage_linux:
+    name: windows-cp${{ matrix.python-version }}-${{ matrix.OPTIONS_NAME }}
+    runs-on: windows-latest
+
+    strategy:
+      # Ensure that a wheel builder finishes even if another fails
+      fail-fast: false
+      matrix:
+        include:
+          - python-version: "3.10"
+            MINIMUM_REQUIREMENTS: 0
+            OPTIONAL_DEPS: 0
+            OPTIONS_NAME: "default"
+          - python-version: "3.11"
+            MINIMUM_REQUIREMENTS: 0
+            OPTIONAL_DEPS: 1
+            OPTIONS_NAME: "optional-deps"
+          - python-version: "3.12"
+            PIP_FLAGS: "--pre"
+            OPTIONS_NAME: "pre"
+          - python-version: "3.13"
+            MINIMUM_REQUIREMENTS: 0
+            OPTIONAL_DEPS: 0
+            OPTIONS_NAME: "default"
+
+    steps:
+      - name: Checkout scikit-image
+        uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version}}
+          allow-prereleases: true
+
+      - name: Install build dependencies
+        env:
+          PIP_FLAGS: ${{ matrix.PIP_FLAGS }}
+        shell: bash
+        run: |
+          source tools/github/setup_build_env.sh
+
+      - name: Build and install from source
+        shell: bash
+        run: |
+          pip install -v --no-build-isolation .
+
+      - name: Install test dependencies
+        env:
+          MINIMUM_REQUIREMENTS: ${{ matrix.MINIMUM_REQUIREMENTS }}
+          OPTIONAL_DEPS: ${{ matrix.OPTIONAL_DEPS }}
+          WITHOUT_POOCH: ${{ matrix.WITHOUT_POOCH }}
+          PIP_FLAGS: ${{ matrix.PIP_FLAGS }}
+        shell: bash
+        run: |
+          source tools/github/setup_test_env.sh
+          mkdir "D:\\tmp"  # used by two doctests (test_PSNR_vs_IPOL, test_gaussian_structural_similarity_vs_IPOL)
+
+      - name: Run tests
+        shell: bash
+        run: |
+          (cd .. && pytest --doctest-plus --showlocals --pyargs skimage)
```

Comment:
From the CI's output it seems like we need 
```
PYTEST: "pytest --config-file ${{ github.workspace }}/pyproject.toml"
```
here as well?

Reply:
Yes, correct; started this branch before that one was merged.